### ISSUE TITLE
document grid_volume in Yee lattice docs

### ISSUE THREAD

**Issue by stevengj** [NanoComp/meep#1268]
In the Yee Lattice section of the manual, we should point out how you can use `sim.gv` (the `grid_volume` object) to find all of these coordates, with the caveat that an `ivec` is `(2i,2j,2k)` compared to the diagram.

**Comment by oskooi** [comment]
Note that an alternative method to obtain the Yee grid coordinates is described in [Field Functions/Coordinates of the Yee Grid](https://meep.readthedocs.io/en/latest/Field_Functions/#coordinates-of-the-yee-grid).

**Comment by stevengj** [comment]
Yes, but I think it would be much better to simply document how you can get it directly since we provide an API for this; using the integration functions for this is awkward.

**Comment by oskooi** [comment]
Which functions from the `grid_volume` class `gv` object should be documented in [Features/Yee Lattice](https://meep.readthedocs.io/en/latest/Yee_Lattice/)? The relevant ones are `vec loc(component, ptrdiff_t index)` and `ivec iloc(component, ptrdiff_t index)` which are demonstrated below. Any others? Perhaps also `ivec little_corner()` and `ivec big_corner()`?

```py
>>> import meep as mp
>>> sim = mp.Simulation(cell_size=mp.Vector3(3,5,0),resolution=10)
>>> sim.init_sim()
-----------
Initializing structure...
time for choose_chunkdivision = 1.00136e-05 s
Working in 2D dimensions.
Computational cell is 3 x 5 x 0 with resolution 10
time for set_epsilon = 0.00346494 s
-----------
>>> sim.gv.nx()
30
>>> sim.gv.ny()
50
>>> sim.gv.nz()
0
>>> sim.gv.iloc(mp.Ex, 11)
<meep.ivec; proxy of <Swig Object of type 'meep::ivec *' at 0x7fa199054090> >
>>> sim.gv.iloc(mp.Ex, 11).x()
-29
>>> sim.gv.iloc(mp.Ex, 11).y()
-28
>>> sim.gv.iloc(mp.Ex, 11).z()
0
>>> sim.gv.loc(mp.Ex, 11)
<meep.vec; proxy of <Swig Object of type 'meep::vec *' at 0x7fa199054090> >
>>> sim.gv.loc(mp.Ex, 11).x()
-1.4500000000000002
>>> sim.gv.loc(mp.Ex, 11).y()
-1.4000000000000001
>>> sim.gv.loc(mp.Ex, 11).z()
0.0
>>> sim.gv.little_corner()
<meep.ivec; proxy of <Swig Object of type 'meep::ivec *' at 0x7fa199054090> >
>>> sim.gv.little_corner().x()
-30
>>> sim.gv.big_corner().x()
30
```

**Comment by smartalecH** [comment]
Is there a function where the user can pass a volume object and a component, and the function returns the corner locations of the component bounding box corresponding to said volume?

**Comment by smartalecH** [comment]
> with the caveat that an `ivec` is `(2i,2j,2k)` compared to the diagram.

![image](https://user-images.githubusercontent.com/24902086/86274082-28631e00-bb9f-11ea-85f8-99c631ba3bc9.png)

Is this true? Doesn't this imply you are "doubly" leapfrogging (as opposed to normal leapfrogging)? Are you sure you didn't mean `(0.5i,0.5j,0.5k)` (i.e. no leapfrogging at all)?
